Core/Core/Courses/CourseListView.swift

```diff
@@ -0,0 +1,175 @@
+//
+// This file is part of Canvas.
+// Copyright (C) 2020-present  Instructure, Inc.
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Affero General Public License as
+// published by the Free Software Foundation, either version 3 of the
+// License, or (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU Affero General Public License for more details.
+//
+// You should have received a copy of the GNU Affero General Public License
+// along with this program.  If not, see <https://www.gnu.org/licenses/>.
+//
+
+import SwiftUI
+import Combine
+
+// swiftlint:disable superfluous_disable_command multiple_closures_with_trailing_closure
+
+@available(iOSApplicationExtension 13.0.0, *)
+public struct CourseListView: View {
+    @Environment(\.appEnvironment) var env: AppEnvironment
+    @Environment(\.viewController) var controller: () -> UIViewController?
+    @ObservedObject var allCourses: Store<GetAllCourses>
+    @State var filter: String = ""
+
+    func configureAppearance() {
+        guard let controller = controller() else { return }
+        UITableView.appearance(whenContainedInInstancesOf: [type(of: controller)]).backgroundColor = .white
```

Comment:
Is a UITableView being used somewhere in here? It's not clear why this would set the background color of courseList

Reply:
It's the backing view of a Form, and this appears to be the only way to set the color. From https://stackoverflow.com/a/58427664/73681

Reply:
This is sad too. But I guess it's better to have it work than not.